src/worker-type.js

```diff
@@ -414,14 +414,15 @@ WorkerType.prototype.createLaunchSpec = function(bid) {
  * function will throw if there is an error found or will return
  * a dictionary of all the launch specs!
  */
-WorkerType.prototype.testLaunchSpecs = function() {
+WorkerType.prototype.testLaunchSpecs = function(awsManager) {
```

Comment:
OK, I'll move those checks.  I'll need to add more checks later for the SubnetId / SGId, so I'll put those in workerTypeCanLaunch, too.